easybuild/easyblocks/c/cuda.py

```diff
@@ -164,12 +164,11 @@ def install_step(self):
         }
 
         # prepare for running install script autonomously
-        qanda = {}
-        stdqa = {
+        qa = [
             # this question is only asked if CUDA tools are already available system-wide
-            r"Would you like to remove all CUDA files under .*? (yes/no/abort): ": "no",
-        }
-        noqanda = [
+            (r"Would you like to remove all CUDA files under .*? (yes/no/abort): ", "no"),
```

Comment:
@branfosj We need to escape special characters here, since this will be compiled via `re.compile`

```suggestion
            (r"Would you like to remove all CUDA files under .*\? \(yes/no/abort\): ", "no"),
```

This was actually also the case with `stdqa`, so perhaps what was there wasn't actually working as intended...